binary-search-trees/01-Binary-Search-Trees.md

```diff
@@ -1,51 +1,95 @@
 # Binary Search Trees
 
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?pid=ceac4982-192f-44a7-88a8-ad91016c972b&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&captions=true&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
+
+## Learning Goals
+
+Students should be able to:
+
+- Compare a binary tree to a linked list
+- Explain how a binary search tree differs from a generic binary tree
+- Write methods to perform the following on a binary search tree:
```

Comment:
I might suggest wording this like "Write and/or comprehend methods to perform..." since the exercise to implement the delete function was removed from this lesson.